### DISCUSSION TITLE
暴露出dotenv的环境变量设置

### DISCUSSION THREAD

**Discussion by Gumplefik** [umijs/umi#9317] [General]
现有骨架核心已包含dotenv，为什么不暴露出更多配置让用户来使用呢？比如典型的分环境配置就需要类似dotenv的支持，而现有的core包貌似只加载了.env和.env.local，是否可以提供支持加载 .env.{variable} 的配置文件？

**Comment by fz6m** [comment]
https://umijs.org/docs/guides/env-variables#umi_env
